module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.cpp

```diff
@@ -226,7 +226,10 @@ namespace module::behaviour::skills {
                           headToBodyRotation = Htc.rotation();
                           orientation        = Eigen::Affine3d(sensors.Htw).rotation().transpose();
                       }
-                      Eigen::Matrix3d headToIMUSpace = orientation * headToBodyRotation;
+
+                      //   Eigen::Matrix3d headToIMUSpace = orientation * headToBodyRotation;
+                      Eigen::Matrix3d headToIMUSpace = headToBodyRotation;
```

Comment:
this seems out of place. why was it changed?

Reply:
Will revert.